util/src/test/java/org/killbill/billing/util/nodes/dao/TestNodeInfoDao.java

```diff
@@ -67,6 +67,15 @@ public void testBasic() throws Exception {
         assertEquals(all.size(), 2);
         assertEquals(all.get(0), newNode1);
         assertEquals(all.get(1), newNode2);
+
+        now = clock.getUTCNow();
+        Thread.sleep(1001);
```

Comment:
Could you just `addDeltaFromReality` instead of sleeping?

Reply:
👍 https://github.com/killbill/killbill/pull/1461/commits/2d1f8aabb29086530fae3e433747c48ecf9c198e